toml.md

```diff
@@ -104,27 +104,39 @@ Keys
 
 A key may be either bare, quoted, or dotted.
 
-**Bare keys** may only contain ASCII letters, ASCII digits, underscores, and
-dashes (`A-Za-z0-9_-`). Note that bare keys are allowed to be composed of only
-ASCII digits, e.g. `1234`, but are always interpreted as strings.
+**Bare keys** may contain any letter-like Unicode character from any Unicode script,
+as well as ASCII digits, dashes and underscores. Punctuation, spaces, arrows, box drawing 
+and private use characters are not allowed.  Note that bare keys are allowed to be
+composed of only ASCII digits, e.g. `1234`, but are always interpreted as strings.
+
+* From the ASCII characters, only A-Z, a-z, 0-9, _ and - are allowed
```

Comment:
* From the ASCII characters, only `A-Z`, `a-z`, `0-9`, `_` and `-` are allowed

Reply:
This little suggestion is still open.

Reply:
@ChristianSi, tx, I just noticed. Fixed now :).